src/app/services/ActivePartsService.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { PartDAO } from '../dao/PartDAO';
+import { IPart, IPartId } from '../domain/icurrentpart';
+import { FirebaseCollectionObserver } from '../dao/FirebaseCollectionObserver';
+import { assert, Utils } from '../utils/utils';
+import { MGPOptional } from '../utils/MGPOptional';
+
+@Injectable({
+    providedIn: 'root',
+})
+/*
+ * This service handles active parts (i.e., being played, waiting for a player, ...),
```

Comment:
i.e. the ones visible in server-page. So it's a complete list :p

Reply:
Je proposais de remplacer ton texte par le miens, ça te semble pas plus précis ?

Reply:
ça fait un peu définition circulaire non ? Car server-page utilise ActivePartsService, donc on s'attendrait à ce que la doc de ActivePartsService dépende pas des composants qui l'utilise non ?

Reply:
Ah ok oui, bah du coup "non finished games", plus accurate.

Reply:
rephrasé